sqlglot/executor/env.py

```diff
@@ -139,6 +139,11 @@ def interval(this, unit):
     return datetime.timedelta(**{unit: float(this)})
 
 
+@null_if_any("this", "expression")
+def arrayjoin(this, expression, null=None):
+    return expression.join(filter(None, (x or null for x in this)))
```

Comment:
you need to check x is none because x could be falsey

Reply:
ha good one yeah I missed it, thanks!

Reply:
Addressed + added `filter_none` helper to get rid of `None` values in iterables. Cleaned up several sections.